Only in hyprland-easymotion: .ccls-cache
diff --unified --show-c-function -ura easymotionorig/easymotionDeco.cpp hyprland-easymotion/easymotionDeco.cpp
--- easymotionorig/easymotionDeco.cpp	2025-01-14 21:58:12.091256542 -0500
+++ hyprland-easymotion/easymotionDeco.cpp	2025-01-14 22:15:03.667233076 -0500
@@ -116,7 +116,7 @@ void CHyprEasyLabel::draw(PHLMONITOR pMo
         return;
 
     const auto PWORKSPACE      = m_pWindow->m_pWorkspace;
-    const auto WORKSPACEOFFSET = PWORKSPACE && !m_pWindow->m_bPinned ? PWORKSPACE->m_vRenderOffset->value() : Vector2D();
+    const auto WORKSPACEOFFSET = PWORKSPACE && !m_pWindow->m_bPinned ? PWORKSPACE->m_vRenderOffset.value() : Vector2D();
 
     const auto ROUNDING = m_iRounding; 
 
@@ -180,15 +180,15 @@ CBox CHyprEasyLabel::assignedBoxGlobal()
 
 		double boxHeight, boxWidth;
 		double boxSize;
-		boxHeight = m_pWindow->m_vRealSize->value().y * 0.10;
-		boxWidth = m_pWindow->m_vRealSize->value().x * 0.10;
+		boxHeight = m_pWindow->m_vRealSize.value().y * 0.10;
+		boxWidth = m_pWindow->m_vRealSize.value().x * 0.10;
 		boxSize = std::min(boxHeight, boxWidth);
-	  double boxX = m_pWindow->m_vRealPosition->value().x + (m_pWindow->m_vRealSize->value().x-boxSize)/2;
-	  double boxY = m_pWindow->m_vRealPosition->value().y + (m_pWindow->m_vRealSize->value().y-boxSize)/2;
+	  double boxX = m_pWindow->m_vRealPosition.value().x + (m_pWindow->m_vRealSize.value().x-boxSize)/2;
+	  double boxY = m_pWindow->m_vRealPosition.value().y + (m_pWindow->m_vRealSize.value().y-boxSize)/2;
     CBox box = {boxX, boxY, boxSize, boxSize};
 
     const auto PWORKSPACE      = m_pWindow->m_pWorkspace;
-    const auto WORKSPACEOFFSET = PWORKSPACE && !m_pWindow->m_bPinned ? PWORKSPACE->m_vRenderOffset->value() : Vector2D();
+    const auto WORKSPACEOFFSET = PWORKSPACE && !m_pWindow->m_bPinned ? PWORKSPACE->m_vRenderOffset.value() : Vector2D();
 
     return box.translate(WORKSPACEOFFSET);
 }
Only in hyprland-easymotion: .git
Only in hyprland-easymotion: hypreasymotion.so
diff --unified --show-c-function -ura easymotionorig/main.cpp hyprland-easymotion/main.cpp
--- easymotionorig/main.cpp	2025-01-14 21:58:12.091256542 -0500
+++ hyprland-easymotion/main.cpp	2025-01-14 22:01:45.821114727 -0500
@@ -239,9 +239,9 @@ APICALL EXPORT PLUGIN_DESCRIPTION_INFO P
 
 
 		g_pGlobalState = std::make_unique<SGlobalState>();
-		HyprlandAPI::addDispatcherV2(PHANDLE, "easymotion", easymotionDispatch);
-		HyprlandAPI::addDispatcherV2(PHANDLE, "easymotionaction", easymotionActionDispatch);
-		HyprlandAPI::addDispatcherV2(PHANDLE, "easymotionexit", easymotionExitDispatch);
+		HyprlandAPI::addDispatcher(PHANDLE, "easymotion", easymotionDispatch);
+		HyprlandAPI::addDispatcher(PHANDLE, "easymotionaction", easymotionActionDispatch);
+		HyprlandAPI::addDispatcher(PHANDLE, "easymotionexit", easymotionExitDispatch);
 		static auto KPHOOK = HyprlandAPI::registerCallbackDynamic(PHANDLE, "keyPress", [&](void *self, SCallbackInfo &info, std::any data) {
 			info.cancelled = oneasymotionKeypress(self, data);
 	});
